pkg/gui/controllers/branches_controller.go

```diff
@@ -165,7 +165,6 @@ func (self *BranchesController) viewUpstreamOptions(selectedBranch *models.Branc
 				ShowBranchHeads:         false,
 			})
 		},
-		Key: 'v',
```

Comment:
I don't have a problem with removing this, I always use `u enter` to invoke it. I wonder though: shouldn't it be possible to override the global key locally? This is a menu, there's no reason to do a range selection there.

Also, I really think we want to disable selecting ranges for menus. Most people probably won't even try, but if you do, it looks confusing, and there's really no reason.

Reply:
I think that 'v' will be for range-select in pretty much all contexts, meaning that if we use it for something else in some situations, it will be confusing, so I think we should avoid using it in any local context for something else.

As for disabling range select in menus, I agree.